### PR TITLE
[DEV-7260] Implement new Covid branding - Covid profile

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**

Update heading on Covid profile page with new image

**Technical details:**
Implementation in Heading.jsx and index.scss

**JIRA Ticket:**
[DEV-7260](https://federal-spending-transparency.atlassian.net/browse/DEV-7260)

**Mockup:**
See ticket link for mocks

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
[N/A] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
[N/A] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
[N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
[N/A ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/_scss/pages/covid19/index.scss [hunk 1]
```diff
@@ -180,6 +180,13 @@
                         padding-top: 2rem;
                         padding-bottom: 1rem;
                         padding: 1rem;
+                        @media(max-width: 559px) {
```
Comment (brett-varney) [inline src/_scss/pages/covid19/index.scss:183]: Is 560 the new 320? Just it isn't one of our "standard" widths.

File: src/_scss/pages/covid19/index.scss [hunk 2]
```diff
@@ -190,7 +197,12 @@
                         letter-spacing: 0;
                         line-height: 2.2rem;
                         text-align: center;
-                        padding: 1rem;
+                        padding: 0 23% 2rem;
+                        @media(max-width: 559px) {
```
Comment (brett-varney) [inline src/_scss/pages/covid19/index.scss:201]: Should 559/560 be one of our width variables?

File: src/js/components/covid19/Heading.jsx [hunk 3]
```diff
@@ -35,6 +34,18 @@ const Heading = () => {
...
+                <img
+                    className="heading__img"
+                    src="img/desktop@2x.png"
+                    alt="An illustration of a masked woman and child holding hands speaking to a masked doctor holding a tablet." />
+            </div>
+            <div style={{ marginLeft: "auto" }} className="heading__img-wrapper">
+                <img
+                    className="heading__img-mobile"
+                    src="img/mobile@2x.png"
+                    alt="An illustration of a masked woman and child holding hands speaking to a masked doctor holding a tablet." />
```
Comment (brett-varney) [inline src/js/components/covid19/Heading.jsx:47]: 508 stuff: purely decorative images shouldn't have alt text, just alt=""

**Review by brett-varney — CHANGES_REQUESTED** [review]
suggest removing alt text, and consider adding 560 to the standard width variables; otherwise very nice!

File: src/js/components/covid19/Heading.jsx (hunk 3 above)
Comment (afnazu) [reply src/js/components/covid19/Heading.jsx]: Got it thanks, removed alt text! 560px was also a specific edge case fix

**Review by brett-varney — APPROVED** [review]

**Pull request merged**